### ISSUE TITLE
Multiple Image Reading

### ISSUE THREAD

**Issue by xyy-Iv** [uber/petastorm#238]
Can anyone show me an example of reading two image at the same time in Pytorch?
The pytorch tutorial did use only make_reader, which contains only one 'data_url'. So my problem is, when using petastorm in semantic segmentation/Image enhancement tasks, how to read multiple data in just one reader?

Any help? Thank you very much!

**Comment by selitvin** [comment]
When working with Petastorm, your dataset should contain the entire sample (data + labels) in a single row in the parquet store. For example, your parquet table should look like this:

| image | segmentation_image |
|-|-|
| image 1 | label_image 1 |
| image 2 | label_image 2 |
| ... | ... |

so when you create a reader:
```python
    with DataLoader(make_reader(dataset_url)) as train_loader:
        sample = next(iter(train_loader))
        image = sample['image']
        labels = sample['label_image']
```
